includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -2923,10 +2923,19 @@ public static function handle_validation_error_update( $redirect_to, $action, $t
 			}
 		}
 
+		if ( $updated_count ) {
+			delete_transient( AMP_Validated_URL_Post_Type::NEW_VALIDATION_ERROR_URLS_COUNT_TRANSIENT );
+		}
+
 		if ( false !== $has_pre_term_description_filter ) {
 			add_filter( 'pre_term_description', 'wp_filter_kses', $has_pre_term_description_filter );
 		}
 
+		// Bail if redirect_is passed as null.
```

Comment:
`redirect_is`?

Reply:
It should be `$redirect_to`. Will update it.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/7614/commits/6d8a72212bfd19e0141e8c3e13ef2bcf4178ddb4